available-hardware.mdx

```diff
@@ -0,0 +1,140 @@
+
+If you are unsure of how many CPU cores you need, it's best to start with either 2 or 4 cores and scale up if you find you need more. For most use cases, 4 cores are sufficient.
+
+## Calculating your required CPU Memory
+
+A guideline for calculating the amount of CPU memory that you require is to use the same amount of CPU memory as you have VRAM. This is because the model will generally be loaded into CPU memory first during initialisation. This happens while the weights are being loaded from storage and the model is being compiled. Once the model is compiled, it is loaded into VRAM and the CPU memory is freed up.  
+Certain libraries, such as _transformers_, do have methods that allow you to use less CPU memory by setting flags such as `low_cpu_mem_usage` which trade off CPU memory for longer initialisation times.
+
+## Storage Requirements
+
+Storage is calculated on an ad-hoc basis and is calculated as you use it.  
+This storage is persistent and will be available to you for as long as you need it.
+
+# Advanced Parameters
+
+This section is for those users that have a large volume of requests and want to optimise their deployments for cost and performance.  
+The following parameters are available to you when deploying your model on Cortex or Conduit.
+
+## Setting your minimum number of instances after a deployment
+
+When a normal deployment starts, we start with a single instance of your deployment. This instance is used to handle all the requests that come in. If the single instance is inadequate, we will scale up your deployment to handle the load automatically.
+This approach allows us to keep your costs as low as possible while still providing you with the performance you need.  
+However, if you have sustained, high-volume (1000s of requests per second) you can quickly see where this approach can be improved.
+
+To tackle this problem, we allow you to set the scale of your deployment using a `min_replicas` parameter.  
+This lets you start your deployment with a minimum number of instances. This means, you skip the scaling up of your deployment and jump straight to the performance you need.
+For example, if you set `min_replicas=4`, your deployment will start with 4 instances and will scale up from there if required.
+
+Note that the number of instances you can have running simultaneously for each of your models is determined by the subscription plan that you are on.
+
+## Calculating your minimum number of instances
+
+If you know the volume of requests you are expecting as well as the amount of time each request takes (in seconds), you can calculate the minimum number of instances you require as follows:
+
+```python
+minReplicas = ceil((requestsPerMin * requestDuration) ÷ 60)
+```
+
+So, if you're receiving 10 000 requests\min and each request takes 0.5s, you can calculate the minimum number of instances you require as follows:
+
+```python
+minReplicas = ceil((10 000 * 0.5) ÷ 60)
+# minReplicas = 84
+```
```

Comment:
should we mention that for a serverless style deployment you can specify 0 min replicas?

Reply:
@jonoirwinrsa should we have that in yet?
Will that be what we're doing at the moment or would that be more like an aws lambda?